src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: {
+    id,
+    title,
+    completed,
+    user,
+  },
+}) => (
+  <article
+    data-id={id}
+    className={classNames('TodoInfo',
+      { 'TodoInfo--completed': completed })}
+  >
+    <h2 className="TodoInfo__title">{title}</h2>
+
+    {user && <UserInfo user={user} key={user.id} />}
```

Comment:
Remove the 'key' prop from the 'UserInfo' component as it's not needed here. Key prop is used when rendering a list of elements.